### ISSUE TITLE
Opening Ur-Ghast loot chest crashes game

### ISSUE THREAD

**Issue by TalisRegis20** [TeamTwilight/twilightforest#1151]
Forge version: 1.16.5 - 36.1.16
Twilight Forest version: 1.16.5 - 4.0.467
Link to crash log: [https://gist.github.com/TalisRegis20/33bb1d9a96ffb55f48d4b6cf49569611#file-crash-2021-05-25_17-56-35-client]

Steps to reproduce:
1.  Defeat Ur-Ghast
2. Open loot chest

What I expected to happen:
Not crash?
What happened instead:
The game crashed.

Some info may be incorrect as I am not entirely sure how to do this.

**Comment by GizmoTheMoonPig** [comment]
Im assuming you have immersive engineering installed? The issue is with the shader that spawns in the loot chest. Its fixed in dev builds, but I completely forgot that wasnt pushed to curseforge yet

**Comment by TalisRegis20** [comment]
Ah ok! Thanks for the quick reply!

**Comment by GizmoTheMoonPig** [comment]
alright, so since a new beta build has been pushed to curseforge and the fix is in it, im gonna close this now. Even though the shader that originally crashed your game is still invalid, it wont crash it anymore as I added a fix to prevent it. All other ur-ghast loot generated after updating will be the correct shader as I fixed the typo I made in the loot table. So just update your game and all will be well.
